example/gardener-local/kind/cluster/templates/_kubeadm_config_patches.tpl

```diff
@@ -13,13 +13,13 @@
       authorization-mode: RBAC,Node
 {{- else }}
       authorization-mode: RBAC,Node,Webhook
-      authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-{{ .Values.environment }}-{{ .Values.networking.ipFamily }}.yaml
+      authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold-{{ .Values.networking.ipFamily }}.yaml
```

Comment:
Maybe you can remove the "skaffold" infix from the filename now

Reply:
Yes make sense, update it.